### PR TITLE
Use bash with source

### PR DESCRIPTION
**Pull request by shaardie**
**Description of the change:**

This patch calls the commands with `source` with the bash directly.

**Motivation for the change:**

This way these commands also work on OS like Debian where `/bin/sh` is actually
not the bash, but another maybe not compatitible shell like `dash`.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [X] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: website/content/en/docs/building-operators/golang/references/envtest-setup.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: manifests generate fmt vet
         mkdir -p ${ENVTEST_ASSETS_DIR}
         test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
-        source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+        base -c "source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out"
```
Comment (jmrodri) [inline website/content/en/docs/building-operators/golang/references/envtest-setup.md:16]:
```suggestion
        bash -c "source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out"
```

**Comment by jmrodri** [comment]
@varshaprasad96 could you verify this change also works on a Mac?

File: website/content/en/docs/building-operators/golang/references/envtest-setup.md (hunk 1 above)
Comment (shaardie) [reply website/content/en/docs/building-operators/golang/references/envtest-setup.md]: Fixed.

**Comment by shaardie** [comment]
> @varshaprasad96 could you verify this change also works on a Mac?

I am sorry, but I have no access to a Mac.

**Comment by estroz** [comment]
This will be fixed by https://github.com/kubernetes-sigs/kubebuilder/pull/2149 upstream then pulled into operator-sdk. If the fix is not accepted upstream, I'd like to port those changes here.

**Comment by jmrodri** [comment]
Upstream kubebuilder PR [2149](https://github.com/kubernetes-sigs/kubebuilder/pull/2149) merged. We've pulled in kubebuilder v3.0.0 in PR #4835 . I'm going to close this PR in favor of the kubebuilder.

**Pull request closed without merging**